CAFAna/Systs/AnaSysts.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include "CAFAna/Core/ISyst.h"
+#include "CAFAna/Systs/Systs.h"
+
+class TH1D;
+
+extern size_t NFluxParametersToAddToStatefile;
+
+namespace ana
+{
+
+	std::vector<std::string> SplitString(std::string input, char delim = ' ');
```

Comment:
It looks like this file is indented with tabs? Please use two spaces to match other code.